Source/src/components/ComponentCamera.h

```diff
@@ -12,6 +12,15 @@ namespace Hachiko
     class ComponentCamera : public Component
     {
     public:
+        enum class CameraType
+        {
+            NONE,
```

Comment:
Could we have a camera with no type at all? What is the use case of NONE type?

Reply:
The case NONE was as default when creating a new component camera, reviewing it with Bernat  he told me to use STATIC for default. So we can remove that NONE type, thank you!

Reply:
Ok!